crmsh/constants.py

```diff
@@ -481,4 +481,8 @@
     }
 }
 
+
+qdevice_help_info = """  QDevice is a separate quorum device which acts as a third-party 
+  arbitration device for the cluster. Its primary use is to allow 
+  a cluster to sustain more node failures than standard quorum rules allow."""
```

Comment:
I'd make it even more verbose telling users it's especially useful for 2-node cluster as the description in the man page does. How about something like:

"QDevice participates in quorum decisions. With the assistance of a third-party arbitrator Qnetd, it provides votes so that a cluster is able to sustain more node failures than standard quorum rules allow. It is recommended for clusters with an even number of nodes and highly recommended for 2 node clusters."

?